lightning/mydump/loader_test.go

```diff
@@ -20,9 +20,11 @@ import (
 	"testing"
 
 	. "github.com/pingcap/check"
+	"github.com/pingcap/tidb-tools/pkg/filter"
```

Comment:
`filter "github.com/pingcap/tidb-tools/pkg/table-filter"  `
should import table-filter?